packages/zoe/src/contractSupport/durability.js

```diff
@@ -39,23 +36,6 @@ export const makeEphemeraProvider = init => {
 };
 harden(makeEphemeraProvider);
 
-/**
- * @deprecated use Recorder getStoragePath() which memoizes
- *
- * @param {import('@agoric/vat-data').Baggage} baggage
- */
-export const makeStorageNodePathProvider = baggage => {
```

Comment:
a breaking chore seems a little funky. I wonder if this is really a fix... if `makeStorageNodePathProvider` broke some rule(s) and hence was buggy. not critical